{{cookiecutter.project_slug}}/tests/CI_docker/context/environment_{{cookiecutter.project_slug}}.yml

```diff
@@ -2,52 +2,45 @@
 # FileType: CONFIG
 # FileCopyrightText: {% now 'utc', '%Y' %}, {{ cookiecutter.full_name }} at GFZ Potsdam
 
-
 name: {{ cookiecutter.project_slug }}
 
 channels: &id1
     - conda-forge
 
 dependencies:
-  - python=3.*.*
-  - pip
-  # test requirements
-  - pytest
-  - pytest-cov
-  - urlchecker
-  - requests
-  - gevent
-  # lint requirements
-  - flake8
-  - pycodestyle
-  - pydocstyle
-  - pylint
-  # doc requirements
-  - sphinx>=4.1.1
-  - sphinx-autodoc-typehints
-  - sphinx-argparse
-  - sphinx_rtd_theme
-  - sphinx-autodoc-typehints
-  - sphinxcontrib-openapi
-  - sphinxcontrib-websupport
-  # deployment requirements
-  - python-build
-  - twine
-  # lint requirements
-  - pre-commit
-  # module requirements
-  - fastapi
-  - geojson
-  - geopandas
-  - httpx
-  - numpy
-  - python-dotenv
-  - rasterio
-  - shapely==1.8.2
-  - uvicorn
+    - python=3.*.*
```

Comment:
Should we then say, 3.12.* since we are pinning it for the pre-commit?

Reply:
As it not decided yet, I left it like this because pre-commit configuration is not clear to me.
What do you think?